project/Dependencies.scala

```diff
@@ -116,14 +116,14 @@ object Dependencies {
     // FIXME: For now TCK protos have been copied and adapted into this project.
     //        Running the TCK is still meaningful as it runs the TCK check against the defined framework version.
     //        Eventually, the final form of protos from should be backported to the framework.
-    //        See https://github.com/lightbend/akkaserverless-java-sdk/issues/605
-    //  akkaslsTckProtocol % "protobuf-src",
-    //  "com.akkaserverless" % "akkaserverless-tck-protocol" % AkkaServerless.FrameworkVersion % "protobuf-src",
+    //        See https://github.com/lightbend/kalix-jvm-sdk/issues/605
+    //  kalixTckProtocol % "protobuf-src",
+    //  "io.kalix" % "kalix-tck-protocol" % Kalix.FrameworkVersion % "protobuf-src",
```

Comment:
Would be good to rename Framework to Proxy, but that should perhaps be a separate PR